tumblr-photo-video-ripper.py

```diff
@@ -196,7 +198,9 @@ def _download_media(self, site, medium_type, start):
                 break
 
             try:
-                data = xmltodict.parse(response.content)
+                xml_cleaned = re.sub(u'[^\x20-\x7f]+',
```

Comment:
Testing pass. You are right. \n \t \r will not effect the downloading result.
Thanks